config/locales/en/course/admin/assessment_settings.yml

```diff
@@ -10,3 +10,4 @@ en:
           submit: 'Update Assessment Settings'
           title: 'Title'
           parent_category: 'Category'
+          enable_public_test_cases_output: 'Enable Public Test Cases Output'
```

Comment:
"Show output for public test cases to students" is a better description of what the option does.

Reply:
Will update code with the description you have suggested. Thanks

Reply:
Also `show_public_test_cases_output` would be a better name 😄